### PR TITLE
cv uptodate

### PR DESCRIPTION
**Pull request by FranDRoss**

### CODE REVIEW

File: staff/francesc-destruels/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        <a href="#Basics">Basics</a>
+        <a href="#Formation">Formation</a>
+        <a href="#Profesional">Profesionals</a>
+        <a href="#Lenguages">Lenguages</a>
+        <a href="#About">About</a>
+    </nav>
+    <section>
+        <article>
+            <h3 id="Basics">Basics</h3>
+            <b>Full Name:</b> Francesc Destruels Rossello<br>
```
Comment (robert-z) [inline staff/francesc-destruels/cv/index.html]: `<b>` is not a semantic tag 😄

File: staff/francesc-destruels/cv/index.html [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+                    Carrefour, S.A
+                </td>
+                <td>
+                    Cashier
+                </td>
+                <td>
+                    Badalona
+                </td>
+                <td>
+                    2008-09
```
Comment (robert-z) [inline staff/francesc-destruels/cv/index.html]: You can use [`<time>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time) tag to  represent a specific period in time.

File: staff/francesc-destruels/cv/index.html (hunk 1 above)
Comment (robert-z) [reply staff/francesc-destruels/cv/index.html]: More info in the following [link](https://stackoverflow.com/questions/271743/whats-the-difference-between-b-and-strong-i-and-em)

Comment (FranDRoss) [reply staff/francesc-destruels/cv/index.html]: I will change it to <strong>

**Pull request merged**